services/stats/stats.go

```diff
@@ -67,6 +68,27 @@ func Init() {
 
 type Tags map[string]string
 
+// Strings returns all key value pairs as an ordered list of strings, sorted by increasing key order
+func (t Tags) Strings() []string {
+	res := make([]string, 0, len(t)*2)
```

Comment:
```suggestion
func (t Tags) Strings() []string {
	if len(t) == 0 {
		return nil
	}
	res := make([]string, 0, len(t)*2)
```

Reply:
OK it makes sense. I would update the PR description though, this makes it more clear :+1:

Reply:
open to proposals :)

Reply:
Done.

Reply:
Also you didn't comment on the suggestion, do you think it doesn't make sense to check the length first? In case it's empty it saves us a few invocations.

Reply:
To be frank I do not consider it critical, but will do so :)